plugins/repository-s3/qa/amazon-s3/build.gradle

```diff
@@ -39,16 +39,37 @@ forbiddenApisTest {
 
 boolean useFixture = false
 
-String s3AccessKey = System.getenv("amazon_s3_access_key")
-String s3SecretKey = System.getenv("amazon_s3_secret_key")
-String s3Bucket = System.getenv("amazon_s3_bucket")
-String s3BasePath = System.getenv("amazon_s3_base_path")
-
-if (!s3AccessKey && !s3SecretKey && !s3Bucket && !s3BasePath) {
-    s3AccessKey = 's3_integration_test_access_key'
-    s3SecretKey = 's3_integration_test_secret_key'
-    s3Bucket = 'bucket_test'
-    s3BasePath = 'integration_test'
+// We test against two repositories, one which uses the usual two-part "permanent" credentials and
+// the other which uses three-part "temporary" or "session" credentials.
```

Comment:
Hopefully we'll split this into two QA projects in a follow up PR, so for now I'd prefer to not rename the existing env vars. 

I understand the permanent/temporary renaming for repositories or buckets but I still think that the env vars names should reflect the feature implemented, ie keeping `amazon_s3_access_key` for the default use case, and maybe `amazon_s3_access_key_for_session_tokens`.

Reply:
I have reverted the changes to the names of the existing environment variables.